packages/wonder-blocks-grid/components/row.js

```diff
@@ -1,14 +1,13 @@
 // @flow
 import * as React from "react";
 import {View, MediaLayoutWrapper} from "@khanacademy/wonder-blocks-core";
+import type {MediaSize, MediaSpec} from "@khanacademy/wonder-blocks-core";
 
 import styles from "../util/styles.js";
 import {matchesSize} from "../util/utils.js";
 import FixedWidthCell from "./fixed-width-cell.js";
 import Gutter from "./gutter.js";
 
-import type {MediaSize, MediaSpec} from "@khanacademy/wonder-blocks-core";
```

Comment:
Interesting that the type imports have to be at the top - it's just clustering based on where the import is coming from. I think I'm ok with this.